MoqTest.cs

```diff
@@ -23,101 +23,91 @@ public void Setup_ExtraCall_Strict ()
               "IMyInterface.DoTheThing(\"C\") invocation failed with mock behavior Strict.\nAll invocations on the mock must have a corresponding setup."));
 
       myMock.Verify();
-      // Strict mocks do not fail in Verify() if not all calls have been set up.
+      // Strict mocks do not fail in Verify() if not all calls have been set up. They fail at the call site.
```

Comment:
please update the assert exception methods and remove calls after the first exception.